src/calculation/ClauseResultsHelpers.ts

```diff
@@ -194,6 +207,29 @@ export function findAllLocalIdsInSort(
   })();
 }
 
+/**
+ * Checks if an operand in the ELM Binary Expression is of type Literal and if so,
+ * pushes its localId and the overall statement localId to emptyResultClauses
```

Comment:
Might want to add to the comment what we are trying to accomplish. Perhaps something like.
> Due to a quirk of the translator as of version 2.??, the comparison annotation includes the Literal and is marked as the localId of the Literal making it look like the comparison always passes. This function adds a mapping to say we should take the result of the comparison clause as the result of the Literal clause as well. This will make the highlighting of the comparison AND literal follow the result of the comparison.